src/trip/components/tripsModal.jsx

```diff
@@ -80,25 +113,27 @@ const TripsModal = ({ mode, initialData, disabled, floated, primary, isBack }) =
                             <Input type='number' min='1' value={inputs.seats} onChange={(e) => setInputs({ ...inputs, seats: e.target.value })}/>
                         </Form.Field>
                     </Form.Group>
-                    <Form.Field required>
-                        <label>Passagers</label>
+                    <Form.Field>
+                        <label>Passagers</label>{
+                        users ?
                         <Dropdown
                             placeholder='Selection des passagers'
                             fluid
                             multiple
                             search
                             selection
                             clearable
-                            options={DATA}
+                            options={users.filter(user => user.username !== login).map(user => ({ value: user._id, text: user.username }))}
                             onChange={(e, { value }) => setInputs({ ...inputs, passengers: value })}
                             value={inputs.passengers}
                         />
+                        : null }
                     </Form.Field>
                 </Form>
                 <Message attached header={error.header} content={error.content} hidden={error.isHidden} error size="mini"/>
             </Modal.Content>
             <Modal.Actions>
-                <Button primary size='mini' onClick={handleSubmit} disabled={!btnEnabled}>
+                <Button primary size='mini' onClick={() => handleSubmit()} disabled={!btnEnabled}>
```

Comment:
I guess it could stay `onClick={handleSubmit}` :)

Reply:
It is exact, i was testing something, i ll change it back

Reply:
Done